content/docs/user-guide/basic-concepts/dvc-cache.md

```diff
@@ -3,7 +3,7 @@ name: 'DVC Cache'
 match: ['DVC cache', cache, caches, cached, 'cache directory']
 ---
 
-The DVC cache is a hidden storage (by default located in the `.dvc/cache`
-directory) for files that are tracked by DVC, and their different versions.
-Learn more about it's
+The DVC cache is a hidden storage (by default in `.dvc/cache`) for files and
+directories tracked by DVC, and their different versions. Data is cached into a
```

Comment:
it's sound ambiguous and not clear why it's needed

Reply:
Oh. I just thought that an sentence that actually gives some info (causes curiosity) AND links to the structure doc was better than "Learn more about it's [structure]."

Why ambiguous?

Reply:
BTW what I'm trying to let the user know is that the cache doesn't preserve the directory structure in the workspace. I guess I could say it like that, is it clearer?

Reply:
`into a special flattened structure` - it can be interpreted in so many ways. If saw this, I would start thinking that it's something very special, complicated, etc.

 > doesn't preserve the directory structure in the workspace.

it's an implementation detail - today it doesn't, tomorrow it will. I would not emphasize it here tbh.

Reply:
> I would not emphasize it here tbh.

OK, rolled back.

Or did you mean remove that sentence completely? I could call it "internal" instead of "hidden" and link that word to the doc where the structure is explained instead.

Reply:
I guess, my biggest problem is `special flattened structure` (not hidden or not)

Reply:
OK np that term is already gone 🙂

Reply:
👍